projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-documents/documenten-api-documents.component.html

```diff
@@ -173,7 +153,12 @@
 
 <ng-template #uploadButton let-disabled="disabled">
   <button
-    [disabled]="(uploadProcessLinkedSet && !uploadProcessLinked) || showZaakLinkWarning || disabled"
+    [disabled]="
+      (uploadProcessLinkedSet && !uploadProcessLinked) ||
+      showZaakLinkWarning ||
+      disabled ||
+      !(canCreateResource$ | async)
```

Comment:
It's a linting issue

```suggestion
      (canCreateResource$ | async) === false
```